src/js/components/Toolbar/Toolbar.tsx

```diff
@@ -87,14 +87,16 @@ const Toolbar = ({
             <div className="u-posAbsolute Toolbar-flyout">{accountBody}</div>
           )}
         </div>
-        <div>
-          <Button type="link" style={btnStyle} onClick={() => setActiveMenu(Menu.Admin)}>
-            {showAdmin && <Icon className="Icon u-spaceLeftLg" name="wrench" style={iconStyle} />}
-          </Button>
-          {activeMenu === Menu.Admin && (
-            <div className="u-posAbsolute Toolbar-flyout">{adminBody}</div>
-          )}
-        </div>
+        {showAdmin && (
```

Comment:
Might need to check if it is a ghost inspector user also?

Reply:
How do I check for ghost inspector user?

Reply:
@jondauz hmmm never mind, this is something you can check in teamsnap spas

Reply:
Yeah, as this is open source and available to people outside if TeamSnap, we don't want to check for anything specific to what we use at TeamSnap. If a conditional needs to be available to a component where we need data like that, it needs to be available via a prop, which it is with the showAdmin prop. In fact, the implementation we have currently might even be too specific, but we can worry about that another time.